Examples/Physics_applications/plasma_acceleration/README.rst

```diff
@@ -46,17 +66,29 @@ For `MPI-parallel <https://www.mpi-forum.org>`__ runs, prefix these lines with `
          :language: ini
          :caption: You can copy this file from ``Examples/Physics_applications/plasma_acceleration/inputs_3d_boost``.
 
-Analyze
--------
+Visualize
+---------
 
-.. note::
+The plasma has a density profile that consists of a long up-ramp followed by a uniform plateau.
 
-   This section is TODO.
+With this configuration the driver excites a nonlinear plasma wake and drives the bubble depleted of plasma electrons where the beam accelerates, as can be seen in Fig
+.. figure:: https://user-images.githubusercontent.com/10621396/290962794-7bdbbff7-39fa-4d2e-8b5f-2030f811d257.png
 
+   :alt: Electron beam, in black, accelerated by wakefield, in red-blue colormap, driven by drive electron beam, in purple.  Set max_steps to 800 in WarpX simulation to generate data for this figure.
 
-Visualize
----------
+   Electron beam, in black, accelerated by wakefield, in red-blue colormap, driven by drive electron beam, in purple.  Set ``max_steps`` to 800 in a WarpX simulation to generate data for this figure.
 
-.. note::
+The beam increases in energy as it propagates in the wake, as shown in Fig.
+
```

Comment:
Add something like this here to reference this figure:
.. _fig_electron_beam_mean_energy_boosted_frame\:

Reply:
done, thanks!